### PR TITLE
renovate: wagtail < 4.2

### PR DESCRIPTION
**Pull request by goapunk**
stay on LTS

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -32,6 +32,10 @@
       {
         "matchPackageNames": ["Django"],
         "allowedVersions": "=3.2"
+      },
+      {
+        "matchPackageNames": ["wagtail"],
+        "allowedVersions": "< 4.2"
```
Comment (fuzzylogic2000) [inline renovate.json:38]: Yeah! But can we go for one version? So, either here "=4.1" or above also "<3.3"

Comment (goapunk) [reply renovate.json:38]: I initially went for `=4.1` but then was unsure since we are still on `4.0` and wouldn't get updates for that.

**Review by fuzzylogic2000 — APPROVED** [review]
Cool!

File: renovate.json (hunk 1 above)
Comment (fuzzylogic2000) [reply renovate.json]: Oh sorry, only saw that now. Yes, that makes sense. Let's change both!

**Pull request merged**